app/controllers/restaurants_controller.rb

```diff
@@ -3,6 +3,11 @@ class RestaurantsController < ApplicationController
 
   def index
     @restaurants = Restaurant.all
+    if params[:query].present?
+      @restaurants = Restaurant.search_by_address(params[:query])
```

Comment:
I believe we should try :
 @restaurants = Restaurant.where("address ILIKE ?", "%#{params[:query]}%")

That way we'll get more results ; for example if you search "otre" right now we don't get any results but we should see any addresses with "notre"